mdn/helpers.py

```diff
@@ -2,14 +2,45 @@
 
 This file is loaded by jingo and must be named helpers.py
 """
-
+from django.utils.six.moves.urllib_parse import urlencode
 from jinja2 import contextfunction, Markup
 from jingo import register
 
-from webplatformcompat.helpers import add_query_param
 from .views import can_create, can_refresh
 
 
+@register.function
+@contextfunction
+def add_filter_to_current_url(context, name, value):
+    """Add a filter to the current URL"""
+    query_parts = []
+    added = False
+    for qs_name, qs_value in context['request'].GET.items():
+        if qs_name == name:
+            added = True
+            query_parts.append((name, value))
+        else:
+            query_parts.append((qs_name, qs_value))
+    if not added:
+        query_parts.append((name, value))
+    return context['request'].path + "?" + urlencode(query_parts)
+
+
+@register.function
+@contextfunction
+def drop_filter_from_current_url(context, name):
+    """Drop a filter from the current URL"""
+    query_parts = []
+    for qs_name, qs_value in context['request'].GET.items():
+        if qs_name != name:
+            query_parts.append((qs_name, qs_value))
+    path = context['request'].path
+    if query_parts:
+        return path + "?" + urlencode(query_parts)
+    else:
+        return path
+
+
```

Comment:
I'm using ``urlencode`` from ``urlparse``.  I tried using the other functions, but you need to start with a full URL (which Django's ``request`` doesn't give me) to use ``urlsplit``/``urlunsplit``